src/main/java/org/dependencytrack/persistence/QueryManager.java

```diff
@@ -1883,7 +1883,7 @@ public ComponentMetaInformation getMetaInformation(UUID uuid) {
         PreparedStatement preparedStatement = null;
         String queryString = """
                 SELECT "C"."ID", "C"."PURL", "IMC"."LAST_FETCH",  "IMC"."PUBLISHED_AT", "IA"."INTEGRITY_CHECK_STATUS" FROM "COMPONENT" "C"
-                JOIN "INTEGRITY_META_COMPONENT" "IMC" ON "C"."PURL" ="IMC"."PURL" JOIN "INTEGRITY_ANALYSIS" "IA" ON "IA"."COMPONENT_ID" ="C"."ID"  WHERE "C"."UUID" = ?
+                JOIN "INTEGRITY_META_COMPONENT" "IMC" ON "C"."PURL" ="IMC"."PURL" LEFT JOIN "INTEGRITY_ANALYSIS" "IA" ON "IA"."COMPONENT_ID" ="C"."ID"  WHERE "C"."UUID" = ?
```

Comment:
We now also need a `null` check here:

https://github.com/DependencyTrack/hyades-apiserver/blob/8bb3729b90da11abf9f2c4eaec2005f15409276d/src/main/java/org/dependencytrack/persistence/QueryManager.java#L1903-L1905

`IntegrityMatchStatus.valueOf(resultSet.getString("INTEGRITY_CHECK_STATUS"))` will throw when no `INTEGRITY_CHECK_STATUS` is returned.

Reply:
yes, added.